litmus/preSi.ml

```diff
@@ -2175,6 +2192,14 @@ module Make
         O.o "static int feature_check(void) {" ;
         if do_self then
           O.oi "cache_line_size = getcachelinesize();" ;
+        if Cfg.variant Variant_litmus.Pac then begin
+          O.fi "if (!check_pac_variant(%S)) return 0;" doc.Name.name;
+          let expect_fpac =
+            if Cfg.variant Variant_litmus.FPac then "true" else "false" in
```

Comment:
Hi @RemyCiterin, during my experiments I found that **stdbool.h** may not be included. May I suggest this change:
```suggestion
            if Cfg.variant Variant_litmus.FPac then "1" else "0" in
```